src/components/Link/ButtonLink/__tests__/ButtonLink.spec.jsx

```diff
@@ -16,6 +16,12 @@ describe('ButtonLink', () => {
     return link.find('a')
   }
 
+  const doShallow = (overrides = {}) => {
+    const button = mount(<ButtonLink {...overrides}>Submit</ButtonLink>)
+
+    return button
+  }
+
```

Comment:
Is this `doShallow` still used? Why was it added if its not used? If it is used, it is called `doShallow` but it does a `mount`..... something is awry here.....